emscripten.py

```diff
@@ -1541,9 +1541,9 @@ def create_basic_funcs(function_table_sigs, invoke_function_names):
   if shared.Settings.EMTERPRETIFY:
     basic_funcs += ['abortStackOverflowEmterpreter']
   if shared.Settings.SAFE_HEAP:
-    if asm_safe_heap():
+    if asm_safe_heap() or shared.Settings.WASM:
       basic_funcs += ['segfault', 'alignfault', 'ftfault']
```

Comment:
`ftfault` is only needed for asm.js (it's for asm.js function table masking)

Reply:
I see.  Its also need to asm2wasm though, right?  I've updated the PR and added a comment.